tox.ini

```diff
@@ -42,11 +42,10 @@ commands=
 deps = -r{toxinidir}/requirements-dev.txt
     coincurve
     database: leveldb
+    trinity: leveldb
 basepython =
     py35: python3.5
     py36: python3.6
-extras =
-    trinity: trinity
```

Comment:
Huh, it's annoying that this didn't work as expected. The extras-style setup ends up being a lot more portable, with less repeated requirements in configs. I don't have any idea why this failed though.

Reply:
Yeah, I was excited about this.  didn't futz with it for long so maybe re-visit at another time (newer tox version?? dunno)